src/consensus/aft/test/logging_stub.h

```diff
@@ -219,7 +219,59 @@ namespace aft
       kv::Tx* tx = nullptr,
       ccf::PrimarySignature* sig = nullptr)
     {
-      return kv::DeserialiseSuccess::PASS;
+      return kv::ApplySuccess::PASS;
+    }
+
+    class ExecutionWrapper : public kv::AbstractExecutionWrapper
+    {
+    public:
+      ExecutionWrapper(const std::vector<uint8_t>& data_) : data(data_) {}
+
+      kv::ApplySuccess execute() override
+      {
+        return kv::ApplySuccess::PASS;
+      }
+
+      kv::ConsensusHookPtrs& get_hooks() override
+      {
+        return hooks;
+      }
+
+      const std::vector<uint8_t>& get_entry() override
+      {
+        return data;
+      }
+
+      Term get_term() override
+      {
+        return 0;
+      }
+
+      kv::Version get_index() override
+      {
+        return 0;
+      }
+      ccf::PrimarySignature& get_signature() override
+      {
+        throw std::logic_error("3. Not Implemented");
```

Comment:
What do the numbers mean?

Reply:
Thanks - removing that it was my debugging :)